<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>inputDialog</class>
 <widget class="QDialog" name="inputDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>661</width>
    <height>202</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文楷体</family>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>监测系统配置设置：</string>
   </property>
  </widget>
  <widget class="QLabel" name="timer_state">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>20</y>
     <width>31</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>20</y>
     <width>101</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>计数倒计时：</string>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>70</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,1">
    <item>
     <widget class="QLabel" name="label">
      <property name="font">
       <font>
        <family>华文楷体</family>
       </font>
      </property>
      <property name="text">
       <string>滑油压力低I值报警：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_oil_i">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>342</x>
     <y>70</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="6,5">
    <item>
     <widget class="QLabel" name="label_5">
      <property name="font">
       <font>
        <family>华文楷体</family>
       </font>
      </property>
      <property name="text">
       <string>滑油压力低II值报警：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_oil_ii">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>341</x>
     <y>110</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="6,0">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="font">
       <font>
        <family>华文楷体</family>
       </font>
      </property>
      <property name="text">
       <string>超速II值报警：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_spd_ii">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,1">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="font">
       <font>
        <family>华文楷体</family>
       </font>
      </property>
      <property name="text">
       <string>超速I值报警值：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="lineEdit_spd_i">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>281</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="1,1">
    <item>
     <widget class="QLabel" name="label_3">
      <property name="font">
       <font>
        <family>华文楷体</family>
       </font>
      </property>
      <property name="text">
       <string>PROFIBUS栈号：</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLineEdit" name="profibus_num">
      <property name="placeholderText">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>150</y>
     <width>301</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="1,1">
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="text">
       <string>启动应用程序</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="etrButton">
      <property name="text">
       <string>默认设置</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
